services/blockchain-indexer/tests/unit/shared/dataservice/business/invoke.test.js

```diff
@@ -174,6 +182,30 @@ describe('invokeEndpoint', () => {
 		});
 	});
 
+	it('should throw error if module or endpoint is not allowed', async () => {
+		jest.mock('../../../../../shared/utils/request', () => ({
+			requestConnector: jest.fn(() => mockBlockByHeightRes),
+		}));
+
+		jest.mock('../../../../../config', () => {
```

Comment:
Might be better if we mocked the Filepath on top so we can change it in one line if needed.
https://github.com/LiskHQ/lisk-service/blob/e1463dddb18559362904855a0356fb1f0ee04259/services/blockchain-indexer/tests/unit/shared/dataservice/business/invoke.test.js#L18